src/main/java/org/opensearch/timeseries/util/ParseUtils.java

```diff
@@ -708,51 +704,28 @@ public static String checkFilterByBackendRoles(User requestedUser) {
      */
     public static boolean shouldUseResourceAuthz(Settings settings) {
         boolean filterByEnabled = AnomalyDetectorSettings.AD_FILTER_BY_BACKEND_ROLES.get(settings);
```

Comment:
the new authz feature is still toggleable by the old setting of `filter_by_backend_role`. However, the ask from AD maintainers is whether this should still be kept as a switch for turning on the new authz scheme?

Reply:
yes I wanted to understand if we should be deprecating this switch already or not. From my understanding is for now:
global flag + ad flag -> new authz route
only ad flag -> current ad authz
no flag -> no sharing concept, default behavior where users can see other configs